src/ConvertsToMilliseconds.php

```diff
@@ -0,0 +1,8 @@
+<?php declare(strict_types=1);
+
+namespace IconLanguageServices\Time;
+
+interface ConvertsToMilliseconds
+{
+    public function inMilliseconds(): float;
```

Comment:
Are we on a 64-bit system? I think on a 64-bit system an INT should be able to represent enough ms for 290 million years.